crontab/ImportPGCatalog.inc

```diff
@@ -105,7 +112,7 @@ class ImportPGCatalog extends BackgroundJob
         //     cache/epub/NNN
         // part, and --directory=$local_catalog_dir so that we end up with
         //     $local_catalog_dir/pgNNN.rdf
-        if ($ret) {
+        if (!$process->isSuccessful()) {
             throw new RuntimeException("Unable to extract files from $local_compressed_file to {$this->local_catalog_dir}");
```

Comment:
It would but we don't have a good place to do that right now beyond `php_errors`. After we get https://github.com/DistributedProofreaders/dproofreaders/issues/1232 in place we can start adding logging throughout the codebase in ways that make sense.